client/swiss-knife/src/helpers.rs

```diff
@@ -83,3 +92,39 @@ pub fn account_address_parser(address: &str) -> AccountAddress {
         })
         .unwrap()
 }
+
+#[derive(Deserialize, Serialize)]
+#[serde(rename_all = "snake_case")]
+pub struct GenerateKeypairResponse {
+    pub private_key: String,
+    pub public_key: String,
+    pub diem_auth_key: String,
+    pub diem_account_address: String,
+}
+
+pub fn generate_key_pair(seed: Option<u64>) -> GenerateKeypairResponse {
```

Comment:
It seems that you moved this function from main.rs to helper.rs and made it public. What is it for? It looks a bit out of place here while all the other Response structs and methods are still in main.rs.